azure-communication-ui/azure-communication-ui-demo-app/src/calling/java/com/azure/android/communication/ui/callingcompositedemoapp/launcher/CallingCompositeKotlinLauncher.kt

```diff
@@ -38,7 +40,8 @@ class CallingCompositeKotlinLauncher(private val tokenRefresher: Callable<String
         showAlert: ((String) -> Unit)?,
     ) {
         initialize(callLauncherActivity.applicationContext)
-        val participantViewData = getParticipantViewData(callLauncherActivity.applicationContext)
+        val
```

Comment:
nitpicking here, you can bring this back to a single line.

Reply:
You're right, fixed in 8853acd